Formula/libgit2-glib.rb

```diff
@@ -27,6 +27,12 @@ class Libgit2Glib < Formula
   depends_on "glib"
   depends_on "libgit2"
 
+  patch do
+    # https://gitlab.gnome.org/GNOME/libgit2-glib/-/merge_requests/29
+    url "https://gitlab.gnome.org/GNOME/libgit2-glib/-/commit/e5e61a523508851daa6e54333c2747b7807496fb.diff"
+    sha256 "6b311f239a872b102a018b1ed14a72fdad18ca4e2c47de3c2b6b2e9e9b66a69d"
+  end
+
   def install
     mkdir "build" do
```

Comment:
I also wonder if we can do without the patch by doing
```ruby
ENV.append "LDFLAGS", "-Wl,-rpath,#{rpath}"
```

Reply:
I’ll try that
I know meson was pretty interesting to figure out

Reply:
ok that worked, lemme push with that instead it seems preferable.

Reply:
@carlocab changed to this method